dbms/src/Common/PODArray.h

```diff
@@ -116,7 +116,7 @@ class PODArrayBase : private boost::noncopyable
 
     void alloc_for_num_elements(size_t num_elements)
     {
-        alloc(roundUpToPowerOfTwoOrZero(minimum_memory_for_elements(num_elements)));
+        alloc(minimum_memory_for_elements(num_elements));
```

Comment:
Plz add some comments to this method. About why don't we use roundUpToPowerOfTwoOrZero.